docs/src/modules/components/HooksApiContent.js

```diff
@@ -76,11 +76,15 @@ import { ${hookName} } from '${source.split('/').slice(0, 2).join('/')}';`}
             language="jsx"
           />
           <span dangerouslySetInnerHTML={{ __html: t('api-docs.importDifference') }} />
-          <Heading text="parameters" hash={`${hookNameKebabCase}-parameters`} level="h3" />
-          <PropertiesTable
-            properties={parameters}
-            propertiesDescriptions={parametersDescriptions}
-          />
+          {Object.keys(parameters).length > 0 && (
```

Comment:
Can we instead show a message that this hook does not accept any input params. People may think it's information missing from the API page. cc @samuelsycamore

Reply:
I agree that would be better. I added it.

Reply:
Oh, but the same is getting applied to hook where the parameters are present but it is not documented yet. Example useSelect - https://deploy-preview-36773--material-ui.netlify.app/base/react-select/hooks-api/#use-select

Reply:
@mnajdova I documented `useSelect` to avoid above issue in https://github.com/mui/material-ui/pull/36773/commits/e91e62a01da9e0d77e8098dcfb7503692c9924b6. However, I wonder why it strictly needs to be an `interface` for it to be documented.

Reply:
I agree, I think it's worthwhile to be explicit that the hook doesn't accept any params. 👍